hugr-core/tests/snapshots/model__roundtrip_cfg.snap

```diff
@@ -21,7 +21,7 @@ expression: "roundtrip(include_str!(\"../../hugr-model/tests/fixtures/model-cfg.
             [%4] [%5]
             (signature (-> [?0] [(adt [[?0]])] (ext)))
             (tag 0 [%4] [%5] (signature (-> [?0] [(adt [[?0]])] (ext))))))
-        (block [%6] [%3 %9]
+        (block [%6] [%3 %6]
```

Comment:
does this change make sense?

Reply:
Yes. This is a test that should have failed but I overlooked it earlier. It's correct now. So this is the test you asked for.

I measured the performance impact and it was very small.